homeassistant/components/renson/entity.py

```diff
@@ -0,0 +1,42 @@
+"""Entity class for Renson ventilation unit."""
+from __future__ import annotations
+
+from renson_endura_delta.field_enum import (
+    DEVICE_NAME_FIELD,
+    FIRMWARE_VERSION_FIELD,
+    HARDWARE_VERSION_FIELD,
+)
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import RensonCoordinator
+from .const import DOMAIN
+
+
+class RensonEntity(CoordinatorEntity):
+    """Renson entity."""
+
+    def __init__(
+        self, name: str, api: RensonVentilation, coordinator: RensonCoordinator
+    ) -> None:
+        """Initialize the Renson entity."""
+        super().__init__(coordinator)
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, "sensor")},
+            manufacturer="Renson",
+            model=api.get_field_value(coordinator.data, DEVICE_NAME_FIELD.name),
+            name="Ventilation",
+            sw_version=api.get_field_value(
+                coordinator.data, FIRMWARE_VERSION_FIELD.name
+            ).split()[-1],
+            hw_version=api.get_field_value(
+                coordinator.data, HARDWARE_VERSION_FIELD.name
+            ),
+        )
+
+        self.api = api
+
+        self._attr_unique_id = f"{name}"
```

Comment:
The name is not an ok entity unique_id.

https://developers.home-assistant.io/docs/entity_registry_index#unique-id-requirements

Reply:
Can I ask why this name variable is not good for a unique value for the attribute?
The name variable contains a unique field name for the defice fields.

Reply:
If more than one config entry is setup, two or more config entries can generate entities with the same field name.

The unique id must be unique within all platform instances of the same domain and integration domain.

Reply:
So if I combile the current field name with for example the mac address this will be unique.

Reply:
Yes, exactly.